src/mod_muc.erl

```diff
@@ -168,7 +168,6 @@ restore_room(ServerHost, Host, Name) ->
 
 forget_room(ServerHost, Host, Name) ->
     LServer = jid:nameprep(ServerHost),
-    ejabberd_hooks:run(remove_room, LServer, [LServer, Name, Host]),
```

Comment:
Maybe add a comment about how we removed this and why?

Reply:
Its in the git history?  But can do

Reply:
Yeah I'm just thinking about whenever we go to upgrade ejabberd, it would be nice to have some context for some of this stuff in the code.

Reply:
Will add.

Reply:
Maybe this is a dumb suggestion since we'd probably be doing it a lot - feel free to ignore.